README.md

```diff
@@ -53,48 +53,53 @@ Mappings that are not overridden this way default to what is found in the `DashD
       "R"                     : ["r"],
       "Ruby"                  : ["ruby", "rubygems", "rails"],
       "Ruby on Rails"         : ["ruby", "rubygems", "rails"],
-      "(HTML) Rails"          : ["ruby", "rubygems", "rails", "html", "svg", "css", "bootstrap", "foundation", "javascript", "jquery", "jqueryui", "jquerym", "angularjs", "backbone", "marionette", "meteor", "moo", "prototype", "ember", "lodash", "underscore", "sencha", "extjs", "knockout", "zepto", "cordova", "phonegap", "yui"],
-      "(JavaScript) Rails"    : ["ruby", "rubygems", "rails", "javascript", "jquery", "jqueryui", "jquerym", "backbone", "marionette", "meteor", "sproutcore", "moo", "prototype", "bootstrap", "foundation", "lodash", "underscore", "ember", "sencha", "extjs", "knockout", "zepto", "yui", "d3", "svg", "dojo", "coffee", "nodejs", "express", "chai", "html", "css", "cordova", "phonegap", "unity3d"],
+      "(HTML) Rails"          : ["ruby", "rubygems", "rails", "html", "svg", "css", "bootstrap", "foundation", "awesome", "statamic", "javascript", "jquery", "jqueryui", "jquerym", "angularjs", "backbone", "marionette", "meteor", "moo", "prototype", "ember", "lodash", "underscore", "sencha", "extjs", "knockout", "zepto", "cordova", "phonegap", "yui"],
+      "(JavaScript) Rails"    : ["ruby", "rubygems", "rails", "javascript", "jquery", "jqueryui", "jquerym", "angularjs", "backbone", "marionette", "meteor", "sproutcore", "moo", "prototype", "bootstrap", "foundation", "lodash", "underscore", "ember", "sencha", "extjs", "knockout", "zepto", "yui", "d3", "svg", "dojo", "coffee", "nodejs", "express", "mongoose", "moment", "require", "awsjs", "jasmine", "sinon", "grunt", "chai", "html", "css", "cordova", "phonegap", "unity3d"],
       "(SQL) Rails"           : ["ruby", "rubygems", "rails"],
       "Ruby Haml"             : ["haml"],
+      "Rust"                  : ["rust"],
       "Sass"                  : ["sass", "compass", "bourbon", "neat", "css"],
-      "Scala"                 : ["scala", "akka", "playscala"],
+      "Scala"                 : ["scala", "akka", "playscala", "scaladoc"],
       "Shell-Unix-Generic"    : ["bash", "manpages"],
       "SQL"                   : ["mysql", "sqlite", "psql"],
       "TCL"                   : ["tcl"],
-      "YAML"                  : ["yaml"]      
+      "TSS"                   : ["titanium"],
+      "TypeScript"            : ["typescript", "javascript", "react", "nodejs", "jquery", "jqueryui", "jquerym", "angularjs", "backbone", "marionette", "meteor", "sproutcore", "moo", "prototype", "bootstrap", "foundation", "lodash", "underscore", "ember", "sencha", "extjs", "knockout", "zepto", "yui", "d3", "svg", "dojo", "express", "mongoose", "moment", "require", "awsjs", "jasmine", "sinon", "grunt", "chai", "html", "css", "cordova", "phonegap", "unity3d", "titanium"],
+      "YAML"                  : ["yaml"],
+      "XML"                   : ["xml","titanium"]
```

Comment:
can you add a missing space after `,` please?

Reply:
you betcha @farcaller! Updated